pkg/server/routes.go

```diff
@@ -44,23 +48,42 @@ func (h StaticHandler) ServeHTTP(ctx *gin.Context) {
 	http.FileServer(h.Assets).ServeHTTP(ctx.Writer, ctx.Request)
 }
 
-// SetupRoutes registers handlers for various URL paths. You can call this
-// function more than once but only the first call will have an effect.
+// SetupRoutes registers handlers for various URL paths.
 func (srv *RegistrationServer) SetupRoutes() error {
 	var err error
 	srv.routesSetup.Do(func() {
+		// initialize default managers
+		keyManager, err := auth.InitializeDefaultKeyManager(srv.Logger(), srv.Config())
+		if err != nil {
+			err = errs.Wrapf(err, "failed to init default key manager: %s", err.Error())
+		}
+		_, err = auth.InitializeDefaultTokenParser(srv.logger, keyManager)
```

Comment:
The default token parser could initialize the default key manager on its own, right? :)

Reply:
No, because the TokenParser might use a different KeyManager. Not yet, sure, but maybe later. TokenParser is completely independent and stateless. I wouldn't pull this into a state unneccessarily. Also, this makes testing easier.

Reply:
> No, because the TokenParser might use a different KeyManager.

A token parser yes. But not the default one.

Reply:
Can you provide a single use case when the default parser would need to use a different key manager?

Also, it’s already stateful since it holds a key manager. I don’t see how using the default manager in the default parser changes it.

Reply:
Ok ok, I give up :-)
I restructured to have the KeyManager stuff not exported. So only TokenParser is used from the outside. See 6f342e0.